hs_core/page_processors.py

```diff
@@ -1,17 +1,24 @@
 """Page processors for hs_core app."""
```

Comment:
Why are we reversing these changes that I had made in cleaning up the page_preprocessor?